docs/app/src/docs.js

```diff
@@ -53,8 +51,8 @@ angular.module('DocsController', [])
    Initialize
    ***********************************/
 
-  $scope.versionNumber = angular.version.full;
-  $scope.version = angular.version.full + '  ' + angular.version.codeName;
+  $scope.versionNumber = NG_VERSION.full;
+  $scope.version = NG_VERSION.full + ' ' + NG_VERSION.codeName;
```

Comment:
The codename for the snapshot is actually "snapshot", at least when built locally.